functorch/test/conftest.py

```diff
@@ -0,0 +1,146 @@
+from _pytest.junitxml import LogXML, _NodeReporter, bin_xml_escape
```

Comment:
Just to check, this is a straight copy of test/conftest.py to make functorch work even though its tests aren't in `test/`?

Reply:
Yeah, exactly. This is the hack that I mention. This file extends pytest capability a bit and it needs to be there on functorch test directory AFAIK